packages/stack/src/views/Stack/CardSheet.tsx

```diff
@@ -36,6 +36,18 @@ export const CardSheet = React.forwardRef<CardSheetRef, Props>(
       const width = document.body.clientWidth;
       const height = document.body.clientHeight;
 
+      // Workaround for mobile Chrome, necessary when a navigation happens
+      // when the address bar has already collapsed, which resulted in an
+      // empty space at the bottom of the page (matching the height of the
+      // address bar). To fix this, it's necessary to update the height of
+      // the DOM with the current height of the window.
+      const vh = window.innerHeight * 0.01;
```

Comment:
can you maybe include the link to the CSS hack website?

Reply:
Yap, added 👍